src/activities/ActivityUsageEntity.js

```diff
@@ -664,10 +664,16 @@ export class ActivityUsageEntity extends Entity {
 			[this.startDate(), activity.dates.startDate],
 			[this.endDate(), activity.dates.endDate],
 			[this.isDraft(), activity.isDraft],
-			[`${this.scoreOutOf()}`, activity.scoreAndGrade.scoreOutOf],
 			[this.inGrades(), activity.scoreAndGrade.inGrades]
 		];
 
+		const scoreOutOf = `${this.scoreOutOf()}`;
+
+		// For activities that don't use scoreOutOf, this.scoreOutOf() is initialized to undefined whereas for those that do it initializes to ''
+		if (scoreOutOf !== 'undefined') {
```

Comment:
This feels quite hacky. The equals method here assumes every activity is using `scoreOutOf` which isn't necessarily the case. If an activity doesn't use `scoreOutOf` then the value of `this.scoreOutOf()` is set to 'undefined' which means the equals check will fail as `activity.scoreAndGrade.scoreOutOf` is an empty string even for activities without `scoreOutOf`

Reply:
Definitely kind of ugly - but i guess it was before too. Would it be better to just modify the `ActivityUsageEntity.scoreOutOf()` method to return an empty string if the scoreOutOf property is falsy?